package.json

```diff
@@ -51,6 +51,7 @@
     "lucide-react": "0.456.0",
     "marked": "14.1.0",
     "nanoid": "5.0.9",
+    "normalize-text": "^2.5.0",
```

Comment:
I don't think we need this dependency. It calls into a browser API (String.prototype.normalize) which meets our browser support targets. IMO we can use it directly.